ts/features/wallet/bancomatpay/component/BPayCard.tsx

```diff
@@ -1,48 +1,37 @@
 import { pipe } from "fp-ts/lib/function";
-import * as O from "fp-ts/lib/Option";
+/* import * as pot from "@pagopa/ts-commons/lib/pot" */ import * as O from "fp-ts/lib/Option";
 import { View } from "native-base";
 import * as React from "react";
-import { Image, ImageStyle, StyleProp, StyleSheet } from "react-native";
+import { Image, StyleSheet } from "react-native";
 import { connect } from "react-redux";
 import bancomatLogoMin from "../../../../../img/wallet/payment-methods/bancomatpay-logo.png";
-import { H3 } from "../../../../components/core/typography/H3";
 import { H4 } from "../../../../components/core/typography/H4";
 import IconFont from "../../../../components/ui/IconFont";
 import I18n from "../../../../i18n";
 import { profileNameSurnameSelector } from "../../../../store/reducers/profile";
 import { GlobalState } from "../../../../store/reducers/types";
+import { IOStyles } from "../../../../components/core/variables/IOStyles";
 import BaseCardComponent from "../../component/card/BaseCardComponent";
-import { useImageResize } from "../../onboarding/bancomat/screens/hooks/useImageResize";
 
-type Props = {
+type OwnProps = {
   phone?: string;
-  bankName: string;
-  abiLogo?: string;
 } & ReturnType<typeof mapStateToProps>;
 
+type Props = ReturnType<typeof mapStateToProps> & OwnProps;
+
```

Comment:
This is redundant, I guess you meant to remove `ReturnType<typeof mapStateToProps>` from `OwnProps` def